service/s3/handwritten-paginators.go

```diff
@@ -0,0 +1,193 @@
+package s3
```

Comment:
style: Filename should use underscore (`_`) not dash (`-`).

Reply:
fixed